### PR TITLE
Removing Overlapping issue in #1464

### PR DESCRIPTION
**Pull request by Siddharth1698**
Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: [Add here what changes were made in this issue and if possible provide links.]

Screenshots for the change:

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:3.3.0'
```
Comment (Masquerade0097) [inline build.gradle]: Not required.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Thu Mar 29 15:52:32 IST 2018
+#Fri Feb 01 02:38:15 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```
Comment (Masquerade0097) [inline gradle/wrapper/gradle-wrapper.properties]: Not required

File: app/src/main/res/layout/main_nav_header.xml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
```
Comment (Masquerade0097) [inline app/src/main/res/layout/main_nav_header.xml]: Indent properly.

File: app/src/main/java/com/eventyay/organizer/core/sponsor/create/CreateSponsorViewModel.java [hunk 4]
```diff
@@ -69,16 +69,17 @@ public void createSponsor() {
...
+
+            compositeDisposable.add(
+                sponsorRepository
+                    .createSponsor(sponsor)
+                    .doOnSubscribe(disposable -> progress.setValue(true))
+                    .doFinally(() -> progress.setValue(false))
+                    .subscribe(createdSponsor -> {
+                        success.setValue("Sponsor Created");
+                        dismiss.call();
+                    }, throwable -> error.setValue(ErrorUtils.getMessage(throwable).toString())));
```
Comment (Masquerade0097) [inline app/src/main/java/com/eventyay/organizer/core/sponsor/create/CreateSponsorViewModel.java]: Not Needed

**Comment by ShridharGoel** [comment]
@Siddharth1698 I'm already working on this as I had mentioned in the issue and a PR is already up which requires some changes. Meanwhile, you can look for other issues.

**Comment by Masquerade0097** [comment]
Closing this.

> @Siddharth1698 I'm already working on this as I had mentioned in the issue and a PR is already up which requires some changes. Meanwhile, you can look for other issues.

**Pull request closed without merging**